package.json

```diff
@@ -23,7 +23,6 @@
     "lint-staged": "lint-staged",
     "lerna": "lerna",
     "packageJsonLint": "npmPkgJsonLint ./packages",
-    "publish-coverage": "mkdir -p ./coverage && node scripts/collect.js && nyc merge ./coverage ./coverage/coverage.json && node scripts/remap.js && cat coverage/lcov.info | coveralls",
```

Comment:
Should we keep this command for reference and pipe to nothing:
```bash
| echo "TODO: send to coveralls with coverallsapp/coverage-reporter"
```
; or should we remove the `scripts` folder altogether?

Reply:
and it enables to remove 1 dependency 🤩

Reply:
we can also remove nyc

Reply:
thanks, I was looking for that one!